waspc/data/Generator/templates/react-app/src/auth/forms/Auth.jsx

```diff
@@ -0,0 +1,300 @@
+{{={= =}=}}
+import React, { useState, useEffect } from 'react'
+import { useHistory } from 'react-router-dom'
+import { createStitches, createTheme } from '@stitches/react'
+
+import { errorMessage } from '../../utils.js'
+import signup from '../signup.js'
+import login from '../login.js'
+import * as SocialIcons from './SocialIcons'
+
+import config from '../../config.js'
+import { styled, css } from '../../stitches.config'
+
+const socialButtonsContainerStyle = {
+  maxWidth: '20rem'
+}
+
+const logoStyle = {
+  height: '3rem'
+}
+
+const Container = styled('div', {
+  display: 'flex',
+  flexDirection: 'column',
+})
+
+const HeaderText = styled('h2', {
+  fontSize: '1.875rem',
+  fontWeight: '700',
+  marginTop: '1.5rem'
+})
+
+const SocialAuth = styled('div', {
+  marginTop: '1.5rem'
+  
+})
+
+const SocialAuthLabel = styled('div', {
+  fontWeight: '500',
+  fontSize: '$sm'
+})
+
+const SocialAuthButtons = styled('div', {
+  marginTop: '0.5rem',
+  display: 'flex',
+
+  variants: {
+    direction: {
+      horizontal: {
+        display: 'grid',
+        gridTemplateColumns: 'repeat(auto-fit, minmax(48px, 1fr))',
+      },
+      vertical: {
+        flexDirection: 'column',
+        margin: '8px 0',
+      }
+    },
+    gap: {
+      small: {
+        gap: '4px',
+      },
+      medium: {
+        gap: '8px',
+      },
+      large: {
+        gap: '16px',
+      }
+    }
+  }
+})
+
+const SocialButton = styled('a', {
```

Comment:
I would maybe put this into a separate component so it can be used with the legacy social buttons?

Reply:
hmm yes, we have to somehow unify the two. Maybe easiest for now to "switch" them all to the new design

Reply:
I spent a bit of time looking into this, but it's not super straightforward, as "new" and "old" buttons don't have the exactly same API - old buttons take images for logos and have text, new ones use SVGs and don't have text, ...

It's nothing complicated and I'm sure we can figure it out but just for the sake of speed I'd rather look at it as a separate issue. I think it won't affect the UX since everybody will mostly be focusing on the new Auth UI form.